OvaryModelGenerator/PelvisModelGenerator.h

```diff
@@ -4,23 +4,24 @@
 #include <iostream>
 
 #include "vtkPolyData.h"
+#include "vtkSmartPointer.h"
 
 class PelvisModelGenerator{
 
-    public:
-        float intercristalDistance;
-        float pelvisScaleFactor;
-        vtkPolyData* pelvisOutputPolydata;
-
     public:
         PelvisModelGenerator();
         ~PelvisModelGenerator();
-        PelvisModelGenerator(const PelvisModelGenerator &other);
+        PelvisModelGenerator(const PelvisModelGenerator &other) = delete;
```

Comment:
It's great to always explicit define copy and assignment operators. But they does not always need to be enabled. We can define them to be deleted which prevent any copy or assignment operations, which are usually safer. 

This case for a filter like class, unless we absolutely need it to be copied or assigned, we can define them to be deleted just to be safe.